doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -63,10 +63,18 @@ nRF9160: Asset Tracker v2
 nRF Machine Learning (Edge Impulse)
 -----------------------------------
 
-* Added non-secure configuration for building :ref:`nrf_machine_learning_app` with :ref:`zephyr:thingy53_nrf5340`.
-* Added secure configuration for building :ref:`nrf_machine_learning_app` with :ref:`zephyr:nrf5340dk_nrf5340`.
-* Added power manager to the :ref:`nrf_machine_learning_app` application.
-* Updated information about custom build types.
+* Added:
+
+  * Non-secure configuration for building :ref:`nrf_machine_learning_app` with :ref:`zephyr:thingy53_nrf5340`.
+  * Secure configuration for building :ref:`nrf_machine_learning_app` with :ref:`zephyr:nrf5340dk_nrf5340`.
+  * Power manager to the :ref:`nrf_machine_learning_app` application.
+
+* Updated:
+
+  * Updated information about custom build types.
```

Comment:
Should we use word `updated` twice here?

Reply:
This is the copy-paste from nrf_desktop. I will leave it for @greg-fer to comment. I will fix if needed.